vck-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidc/OidcSiopVerifier.kt

```diff
@@ -236,6 +240,24 @@ class OidcSiopVerifier(
          * [clientMetadataUrl], that the URL shall point to [OidcSiopVerifier.metadataWithEncryption].
          */
         val encryption: Boolean = false,
+
+        val rqesParameters: RqesParameters? = null,
```

Comment:
Please add documentation.
If only the `RqesOidcVerifier` can handle `TransactionData` as a class, is adding all those RQES parameters here (in the openid module) really appropriate?